pymongo/synchronous/mongo_client.py

```diff
@@ -860,6 +860,10 @@ def __init__(
             # This will be used later if we fork.
             MongoClient._clients[self._topology._topology_id] = self
 
+    def connect(self) -> None:
+        """Explicitly connect synchronously."""
+        self._get_topology()
```

Comment:
We should avoid adding connect() and aclose() to the sync client.

Reply:
What if we rename them to be private? Like `aconnect` -> `_aconnect` and `aclose` -> `_aclose`? There isn't a clean way of having the synchro process just not convert specific methods.

Reply:
How about something like this?:
```python
_IS_SYNC = False
class AsyncMongoClient:
    if not _IS_SYNC:
        async def aclose(self):
            pass
```

Reply:
aclose -> close SGTM. For aconnect() I would prefer the approach I outlined above.

Reply:
Actually. renaming aconnect -> _aconnect is fine too.

Reply:
The synchro process will still convert the `aconnect` method within the `if not _IS_SYNC` block and add it to the sync client source. Is that acceptable?

Reply:
I'm good with any approach as long as aconnect() or connect() isn't a public method on MongoClient.